bedrock/firefox/urls.py

```diff
@@ -129,6 +129,7 @@
     page("firefox/welcome/11/", "firefox/welcome/page11.html", ftl_files=["firefox/welcome/page11"]),
     page("firefox/welcome/12/", "firefox/welcome/page12.html", active_locales=["en-US", "en-CA", "en-GB"]),
     page("firefox/welcome/13/", "firefox/welcome/page13.html", ftl_files=["firefox/welcome/page13"]),
+    page("firefox/welcome/14/", "firefox/welcome/page14.html", ftl_files=["firefox/welcome/page10", "firefox/welcome/page14"]),
```

Comment:
Issue (maybe blocking?): it's a little unusual to be mixing two different FTL files like this. The first file in the array will be used for activation, meaning all locales that have `page10.ftl` translated will be enabled in production without considering `page14.ftl`. This will likely mean some locales end up with mixed language content, where `page10.ftl` is translated, but `page14.ftl` is rendered in English. We could try and use `ftl_has_messages()` to do some conditional rendering, but that might be difficult if there's not a fallback string for everything.

A simpler approach this might be to stick to using just one `page14.ftl` file here, and treat this like a new page. It would mean a handful of duplicate strings between files, but tools like Pontoon support fuzzy string matching to make this less of an issue.

Reply:
TIL that pontoon has fuzzy string matching, ty!